### DISCUSSION TITLE
Internal server error when trying to start containers after first backup

### DISCUSSION THREAD

**Discussion by martinbenson** [nextcloud/all-in-one#4668] [Questions] [answered]
### Steps to reproduce
1. Initiate first backup via AIO interface - it completes successfully
2. Try to start the containers

### Expected behavior 
The containers should start 

### Actual behavior 

It reports the following error:

Type: GuzzleHttp\Exception\ServerException
Code: 500
Message: Server error: DELETE http://localhost/v1.41/containers/nextcloud-aio-database?v=true resulted in a 500 Internal Server Error response:
{"message":"container 647d33f11618c7777fbbad737c10187199601226708228b0103639fb3173a694: driver &quot;zfs&quot; failed to remove (truncated...)
File: /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php
Line: 113
Trace: #0 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Middleware.php(72): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), NULL, Array, NULL)
https://github.com/nextcloud/.github/pull/1 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(209): GuzzleHttp\Middleware::GuzzleHttp{closure}(Object(GuzzleHttp\Psr7\Response))
https://github.com/nextcloud/.github/pull/2 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(158): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), NULL)
https://github.com/nextcloud/.github/pull/3 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/TaskQueue.php(52): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise{closure}()
https://github.com/nextcloud/.github/pull/4 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(251): GuzzleHttp\Promise\TaskQueue->run(true)
https://github.com/nextcloud/.github/pull/5 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(227): GuzzleHttp\Promise\Promise->invokeWaitFn()
https://github.com/nextcloud/.github/pull/6 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(272): GuzzleHttp\Promise\Promise->waitIfPending()
https://github.com/nextcloud/.github/pull/7 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(229): GuzzleHttp\Promise\Promise->invokeWaitList()
https://github.com/nextcloud/.github/pull/8 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(69): GuzzleHttp\Promise\Promise->waitIfPending()
https://github.com/nextcloud/.github/pull/9 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Client.php(189): GuzzleHttp\Promise\Promise->wait()
https://github.com/nextcloud/.github/pull/10 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/ClientTrait.php(129): GuzzleHttp\Client->request('DELETE', 'http://localhos/...', Array)
https://github.com/nextcloud/.github/pull/11 /var/www/docker-aio/php/src/Docker/DockerActionManager.php(157): GuzzleHttp\Client->delete('http://localhos/...')
https://github.com/nextcloud/.github/pull/12 /var/www/docker-aio/php/src/Controller/DockerController.php(60): AIO\Docker\DockerActionManager->DeleteContainer(Object(AIO\Container\Container))
https://github.com/nextcloud/.github/pull/13 /var/www/docker-aio/php/src/Controller/DockerController.php(33): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-d...', true)
https://github.com/nextcloud/.github/pull/14 /var/www/docker-aio/php/src/Controller/DockerController.php(33): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-n...', true)
https://github.com/nextcloud/.github/pull/15 /var/www/docker-aio/php/src/Controller/DockerController.php(211): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-a...', true)
https://github.com/nextcloud/.github/pull/16 /var/www/docker-aio/php/src/Controller/DockerController.php(191): AIO\Controller\DockerController->startTopContainer(true)
https://github.com/nextcloud/.github/issues/17 /var/www/docker-aio/php/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponse.php(38): AIO\Controller\DockerController->StartContainer(Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
https://github.com/nextcloud/.github/pull/18 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(358): Slim\Handlers\Strategies\RequestResponse->__invoke(Array, Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
https://github.com/nextcloud/.github/pull/19 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\Routing\Route->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/20 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/21 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(315): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/22 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(68): Slim\Routing\Route->run(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/issues/23 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(482): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/24 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
https://github.com/nextcloud/.github/pull/25 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/issues/26 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
https://github.com/nextcloud/.github/pull/27 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/issues/28 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
https://github.com/nextcloud/.github/pull/29 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/30 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
https://github.com/nextcloud/.github/issues/31 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/32 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/pull/33 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
https://github.com/nextcloud/.github/issues/34 /var/www/docker-aio/php/public/index.php(185): Slim\App->run()
https://github.com/nextcloud/.github/pull/35 {main}
Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
8.2.1

#### Current channel 
latest

#### Other valuable info 
Everything worked fine before doing the first backup of the instance

**Comment by szaimen** [comment, marked as answer]
> driver "zfs" failed to remove (truncated...)

Hi, this is your issue and it is caused by your docker installation. (And not by aio)

See https://www.google.com/search?q=%22driver+zfs+failed+to+remove%22&sca_esv=2e47e62a151241ca&sca_upv=1&ei=jG1IZurWAY2Gxc8PoraOkAU&oq=%22driver+zfs+failed+to+remove%22&gs_lp=EhNtb2JpbGUtZ3dzLXdpei1zZXJwIh0iZHJpdmVyIHpmcyBmYWlsZWQgdG8gcmVtb3ZlIjIHEAAYgAQYEzIHEAAYgAQYEzIIEAAYExgFGB4yCBAhGKABGMMESOo5UOsMWJI4cAF4AJABAJgBiQGgAYYFqgEDMy4zuAEDyAEA-AEBmAIHoAK8BcICCxAAGLADGBMYBxgewgIKEAAYgAQYsAMYE8ICCxAAGIAEGLADGKIEwgIIEAAYgAQYogTCAggQABgTGAcYHsICChAhGKABGMMEGArCAggQABgTGBYYHsICBRAhGJ8FmAMAiAYBkAYFkgcDMy40oAeLHQ&sclient=mobile-gws-wiz-serp
